README.md

```diff
@@ -130,6 +130,18 @@ Notice 5: The template uses Android Gradle Plugin 8.1.2, for which the minimum J
 
 The script will ask you for a project relative path (like *../my-new-project-dir*). Next up the script will ask some more parameters, answer those to the best of your ability. After that, your new project will be created. To get it to run see Configuring Firebase section below.
 
+The relative path and project name can be passed as command line arguments as well:
+
+   ```bash
+   bundle exec cap android:create <relative_path> <project_name>
+   ```
+
+   ```bash
+   bundle exec cap android:create_library <relative_path> <project_name>
+   ```
+
+This method is used in CI builds but feel free to use it while creating your project as well, the limitation to this however is that you cannot change the package name (`mobi.lab` by default).
```

Comment:
Hmm, I understand why you did it like this, but this limitation makes this practically useless for anything else than the CI.
Maybe makes sense to create a followup JIRA task to also allow / require the package name here? 
Doesn't need to be done now, but then we have this in the backlog

Reply:
Better to just do it. Makes it all explicit

Reply:
Added package to command line arguments as well but made it optional, let me know if you think it should be required.

Reply:
I feel the input part is pretty complicated as it is now + the current implementation allows to pass in only the path argument which is then ignored. 

Seems that a better solution would be to use named arguments which would allow to set all the values individually and if some are missing, they can be asked or generated individually.

A way to do it is to use `-s name=value` to pass in named arguments. And these can be assigned to variables with `variable = fetch(:name, nil)`

So we could end up with running the script with `bundle exec cap android:create -s path=/some/path -s name=MyAwesomeProject -s package=my.awesome.project`